lang/java/reef-tests/src/test/java/org/apache/reef/tests/yarnpreemption/package-info.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * Test reef.io.watcher.
+ */
+package org.apache.reef.tests.yarnpreemption;
```

Comment:
@motus Thanks for your review! I uploaded a design doc that contains a plan for `reef-runtime-local` on JIRA. (https://issues.apache.org/jira/browse/REEF-836)

IMHO, we should separate tests according to the underlying resource manager because preemption scenario depends on the environment.